packages/msgraph/test/Adaptor.test.js

```diff
@@ -1,13 +1,29 @@
 import { expect } from 'chai';
+import { setGlobalDispatcher } from 'undici';
+
 import { execute, getDrive, getFolder, getFile } from '../src/Adaptor.js';
 
 import MockAgent from './mockAgent.js';
 import { fixtures } from './fixtures.js';
 
-import { setGlobalDispatcher } from 'undici';
-
 setGlobalDispatcher(MockAgent);
 
+const readStream = async stream => {
```

Comment:
I've got two problems here.

First off this is a really over-complicated way to resolve the stream. There are much easier ways!

Secondly, and more importantly, these tests are showing us that we've broken `getFile` in the adaptor. Rather than coding around the problem in the unit tests, we should address it properly because it's unusable in job code right now.

Reply:
I have remove this approach in favor of using `parseAs`